### PR TITLE
Unlocked AI Capabilities for the Apex RPG7

### PR DESCRIPTION
**Pull request by G3rrus**
Overriding the AI behaviour for the ammo type R_PG7_F in order to allow them to:

-Attack air assets
-Attack infantry

### CODE REVIEW

File: addons/ai/CfgAmmo.hpp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class CfgAmmo {
+	class RocketCore;
```
Comment (diwako) [inline addons/ai/CfgAmmo.hpp]: Why define `RocketCore` when it is not used?

Comment (G3rrus) [reply addons/ai/CfgAmmo.hpp]: Because rocketBase inherits from rocketCore?

**Comment by G3rrus** [comment]
You were right, tested it without RocketCore and worked just the same.

File: addons/ai/config.cpp [hunk 2]
```diff
@@ -5,10 +5,14 @@ class CfgPatches {
         units[] = {};
         weapons[] = {};
         requiredVersion = REQUIRED_VERSION;
-        requiredAddons[] = {"fpa_main"};
+        requiredAddons[] = {
+            "A3_Weapons_F_Exp",
```
Comment (Cuel) [inline addons/ai/config.cpp]: not needed

**Comment by G3rrus** [comment]
Better now?

**Pull request merged**